Extensions/blacklist.js

```diff
@@ -152,6 +148,10 @@ XKit.extensions.blacklist = new Object({
 
 		$(document).on('click', ".xblacklist_open_post", XKit.extensions.blacklist.unhide_post);
 
+		const postSel = XKit.css_map.keyToCss('listTimelineObject') || '.post';
+		const postContentClasses = XKit.css_map.keyToClasses('post') || ['.post_content'];
+		const blacklistedPostContentSel = postContentClasses.map(cls => `.xblacklist_blacklisted_post ${cls}`).join(', ');
+
 		if (this.preferences.mini_block.value === true) {
```

Comment:
```suggestion
		if (this.preferences.mini_block.value === true && !XKit.page.react) {
```

Reply:
I'll revert if fixing the mini ui is easier than I thought it'd be

Reply:
It was easier than I thought it'd be